docs/jetstream/configuration.md

```diff
@@ -326,3 +328,73 @@ data_source = "search_clients_engines_sources_daily"
 [metrics.urlbar_amazon_search_count.statistics.bootstrap_mean]
 [metrics.urlbar_amazon_search_count.statistics.deciles]
 ```
+
+### Defining Exposure Signals
+
+Many Nimbus features will send a [Nimbus exposure event] automatically when the feature configuration is consulted;
+these are `normandy#expose` events on desktop and `nimbus_events.exposure` events in Glean.
+However, it is also possible to define custom exposure events:
+
+```toml
+[experiment.exposure_signal]
+name = "nimbus"
+friendly_name = "Nimbus exposure signal"
+description = "Nimbus desktop feature exposure signal"
+data_source = "events_unnested"
+select_expression = "
+   event_category = 'normandy'
+   AND event_method = 'expose'
+   AND event_object = 'nimbus_experiment'
+   AND event_string_value = 'experiment-slug'
+   AND normalized_channel = 'release'
+"
+window_start = 0                    # optional
+window_end = "analysis_window_end"  # optional
+
+
+[data_sources.events_unnested]
```

Comment:
Just curious why this is needed. Events isn't one of the predefined data sources?

Reply:
Yep, this is redundant. Removed and using existing `events` instead